src/pages/HomePage/ContactSection.jsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Typography,
+  Box,
+  Link,
+  IconButton,
+  Container,
+  Grid,
+  Stack,
+} from "@mui/material";
+import EmailIcon from "@mui/icons-material/Email";
+import { FaInstagram } from "react-icons/fa";
+
+const ContactSection = () => {
+  return (
+    <Box
+      sx={{
+        backgroundColor: "black",
+        height: "150vh",
```

Comment:
Why is this 150?

Reply:
removed, was actually serving no purpose